pandas/core/arrays/arrow/array.py

```diff
@@ -2192,11 +2192,11 @@ def _str_rstrip(self, to_strip=None):
         return type(self)(result)
 
     def _str_removeprefix(self, prefix: str):
```

Comment:
Does this array implement `_str_removesuffix`?

Reply:
Yes, in the mixin class

Reply:
Ah sounds good. Just a merge conflict otherwise looks good